source/agora/test/ValidatorRecurringEnrollment.d

```diff
@@ -0,0 +1,30 @@
+module agora.test.ValidatorRecurringEnrollment;
+
+import agora.test.Base;
+
+import core.thread.osthread;
+
+import std.stdio;
+
+unittest
+{
+    TestConf conf = { validator_cycle : 3, txs_to_nominate : 1, recurring_enrollment : true };
```

Comment:
Please set the `quorum_threshold` to `100`.

Reply:
We might even consider changing the default to `100` in `TestConf` because it's a bit dangerous that it's implicitly lower, it circumvents expectations in the tests.

Reply:
`validator_cycle` should be a part of `ValidatorConfig`.

Reply:
I believe there is no way to set `validator_cycle`, with `validator_cycle` being removed from `TestConf` right?